packages/x-components/src/components/animations/animate-scale/animate-scale.style.scss

```diff
@@ -0,0 +1,100 @@
+.x-animate-scale {
```

Comment:
I'm worried about the generated size of this file, 22kb of css, which end up being 3kb gzipped just for a simple scale animation.

In theory you should only need 18 frames for the animation to look smooth, 60Hz of average screens multiplied by 0.3, or 36 frames if you still want to support super smooth screens. I tested with 36 and it reduces the css to 10kb and 1kb ggzipped, and still looks smooth.

Reply:
I agree. We are creating a lot of unnecessary frames here.